### PR TITLE
Move to lints in package:lints

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

**Review by johnpryan — APPROVED** [review]

File: extension_methods/example/core_type_extensions.dart [hunk 1]
```diff
@@ -32,6 +32,6 @@ extension Silliest on Object {
   bool notAgain(Object other) => this != other;
 }
 
-extension ExtremelySilly on Null {
+extension ExtremelySilly on void {
```
Comment (natebosch) [inline extension_methods/example/core_type_extensions.dart:35]: Did a lint surface this? I think it's a bug in the lint.

Comment (natebosch) [reply extension_methods/example/core_type_extensions.dart:35]: https://github.com/dart-lang/linter/issues/2680

Comment (mit-mit) [reply extension_methods/example/core_type_extensions.dart:35]: Thanks for filing the bug.

How about I just change this to `extension ExtremelySilly on Object?` for now? The file doesn't have any example with null safety, so would be good to add that, I think

Comment (mit-mit) [reply extension_methods/example/core_type_extensions.dart]: Done

**Pull request merged**